ui.apps/src/main/javascript/minicart/src/components/Checkout/editableForm.js

```diff
@@ -96,7 +115,37 @@ const EditableForm = props => {
                 });
             }
 
-            setPaymentMethodOnCart({ variables: { cartId: cartId, paymentMethodCode: args.paymentMethod.code } });
+            // Store payment method in state, so we can later use it in the
+            // payment step of the checkout.
+            dispatch({ type: 'setPaymentMethod', paymentMethod: args.paymentMethod });
+
+            switch (args.paymentMethod.code) {
+                case 'braintree': {
+                    setBraintreePaymentMethodOnCart({
+                        variables: {
+                            cartId: cartId,
+                            paymentMethodCode: args.paymentMethod.code,
+                            nonce: args.paymentNonce
+                        }
+                    });
+                    break;
+                }
+                case 'braintree_paypal': {
```

Comment:
You can merge this `case` with the first one, the code seems exactly the same.

Reply:
I fixed that 👍

Reply:
Thanks, I fixed that 👍